invenio_rdm_records/services/__init__.py

```diff
@@ -8,7 +8,7 @@
 """High-level API for wokring with RDM records, files, pids and search."""
 
 from .config import RDMFileDraftServiceConfig, RDMFileRecordServiceConfig, \
-    RDMRecordServiceConfig
+    RDMRecordServiceConfig, SecretLinkServiceConfig
```

Comment:
For my comment on the `ext.py`... you are actually exporting it, so there is no need to change the import (in `ext.py`)

Reply:
still present